$MirOS: ports/lang/python/2.3/patches/patch-Modules_socketmodule_c,v 1.3 2005/11/15 19:16:58 tg Exp $
$OpenBSD: patch-Modules_socketmodule_c,v 1.2 2004/08/11 08:11:50 xsa Exp $
--- Modules/socketmodule.c.orig	2007-03-31 20:56:11.000000000 +0200
+++ Modules/socketmodule.c	2008-04-22 23:06:09.000000000 +0200
@@ -73,9 +73,6 @@ Local naming conventions:
 #include "Python.h"
 #include "structmember.h"
 
-#undef MAX
-#define MAX(x, y) ((x) < (y) ? (y) : (x))
-
 /* Socket object documentation */
 PyDoc_STRVAR(sock_doc,
 "socket([family[, type[, proto]]]) -> socket object\n\
@@ -3602,7 +3599,7 @@ socket_inet_aton(PyObject *self, PyObjec
 
 #if !defined(HAVE_INET_ATON) || defined(USE_INET_ATON_WEAKLINK)
 	/* Have to use inet_addr() instead */
-	unsigned long packed_addr;
+	int packed_addr;
 #endif
 	char *ip_addr;
 
